client/src/pages/FloorPlan.js

```diff
@@ -35,7 +41,7 @@ const Desk = ({ desk, booking }) => {
 	);
 };
 
-const FloorPlan = ({ desks, bookings }) => {
+const FloorPlan = ({ desks, bookings, deskId, setDeskId }) => {
```

Comment:
I still think that within the FloorPlan component, these props should be currentlySelectedDeskId/getCurrentlySelectedDeskId, and that it's quite an important aspect of code readability.